test/test_model.py

```diff
@@ -1104,10 +1104,10 @@ def test_storage(self, fake_script: FakeScript, fake_juju_version: None):
         fake_script.write(
             'storage-get',
             """
-            if [ "$2" = disks/0 ]; then
-                echo '"/var/srv/disks/0"'
-            elif [ "$2" = disks/1 ]; then
-                echo '"/var/srv/disks/1"'
+            if [ "$3" = disks/0 ]; then
+                echo '{"kind": "filesystem", "location": "/var/srv/disks/0"}'
```

Comment:
Why does this output change?

Reply:
This is the same as the storage example above. We (a) always use `--format-json` now, and (b) always get all the attributes and let callers access the one(s) they care about.